signal0ne/README.md

```diff
@@ -1 +1,31 @@
-# signal0ne
\ No newline at end of file
+# Signal0ne
+Signal0ne is an ultimate AI debugging and issue resolution tool.
+See the demo at: https://signaloneai.com/#pro
+
+#### Features
+
+- Issue analysis
+
+- Code fix tip(PRO only)
+
+# How to use it
+
+#### Download docker desktop extension to run agent that will discover issues.
+https://open.docker.com/extensions/marketplace?extensionId=signaloneai/signalone-extension
+![Docker Desktop](https://github.com/Signal0ne/signal0ne-vscode-extension/blob/8933d27d0a717ac7d823e877d74833c58bd374cc/signal0ne/resources/docker_desktop_marketplace.png?raw=true)
+
+#### Subscribe to PRO version(it is free for a month without a need to provide payment details)
+
+![PRO](https://github.com/Signal0ne/signal0ne-vscode-extension/blob/c3c776d00236c97c7704b8082e0dcae8bb18cd34/signal0ne/resources/docker_desktop_pro.png?raw=true)
+
+#### Login to both extensions and let it run in the background to discover all issues in your running applications you are developing
+
+![Issues View](https://github.com/Signal0ne/signal0ne-vscode-extension/blob/3fcee544f0ec251166c49e483133a1c73c64ae10/signal0ne/resources/vscode_issues_view.png?raw=true)
+
+#### Add code to context with "Code fix" feature to get proposed solution of issue in your code
```

Comment:
maybe rewrite it to something like this: 
[Just a suggestion] `Select the issue from the list and select the code block that you think can be related to the issue and select the 'Fix code' option from the context menu to try fix the issue using our extension`